internal/factory/daemonset_test.go

```diff
@@ -22,11 +22,21 @@ func TestNewDaemonsetSetsAllLabelsToBeTheSame(t *testing.T) {
 	daemonSet := NewDaemonSet("thenname", "thenamespace", "thecomponent", "thecomponent", podspec)
 
 	expLabels := daemonSet.ObjectMeta.Labels
```

Comment:
Not on this line: The name of the test `TestNewDaemonsetSetsAllLabelsToBeTheSame` does not match the content anymore.